care/facility/models/asset.py

```diff
@@ -177,3 +182,10 @@ class AssetTransaction(BaseModel):
     performed_by = models.ForeignKey(
         User, on_delete=models.PROTECT, null=False, blank=False
     )
+
+
+class AssetService(BaseModel):
+    asset = models.ForeignKey(Asset, on_delete=models.PROTECT, null=False, blank=False)
+
+    serviced_on = models.DateField(default=None, null=True, blank=True)
```

Comment:
Do we need to allow asset service records without a serviced_on datetime for the record?

Reply:
Currently the system allows users to enter a blank date with a note or vice versa, so I carried over that behavior. It doesn't make sense to have a service record without a date, but if it's being used already then, we have to keep the behavior, otherwise we can make the date mandatory.

Reply:
so the timeline table would be sorted by? modified/created date?

Reply:
@gigincg is it fine to allow serviced_on to be blank?

we could just allow null for migration purposes but have `blank=False` to make it a required field in forms.

Reply:
okay, let's do that!